src/sidebar/components/Annotation/AnnotationEditor.tsx

```diff
@@ -93,10 +89,9 @@ function AnnotationEditor({
     /**
      * Remove tag from draft if present.
      *
-     * @param {string} tag
-     * @return {boolean} Tag removed from draft
+     * @return Tag removed from draft
```

Comment:
Huh. This is interesting. You've kicked up an interesting case here! The removal of the redundant type information makes this comment ambiguous. It sounds like the removed  _tag_ (string) itself is returned, when in fact the return is boolean (FWIW the original `@return` comment was a little vague).

One possibility:

```suggestion
     * @return `true` if tag removed from draft, `false` if tag not found in draft tags
```

Other options:

* Remove this comment entirely as the return logic is pretty easy to scan in this very brief callback function.
* Re-instate the (redundant) type annotation. Hey, we have conventions here, but we like to adapt if they don't apply well!

Reply:
My personal take on this, I would just remove the comment.

Comments usually have the opposite effect when the code is clear enough to be understood without them.